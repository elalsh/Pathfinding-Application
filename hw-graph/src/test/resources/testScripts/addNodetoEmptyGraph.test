# The expected value is the for graph to have 1 node

# Create a graph
CreateGraph graph1

# Add node single node n1 to the graph
AddNode graph1 n1

# Print the node in the graph
ListNodes graph1