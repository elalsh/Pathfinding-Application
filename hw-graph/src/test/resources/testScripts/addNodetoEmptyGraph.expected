# The expected value is the for graph to have 1 node

# Create a graph
created graph graph1

# Add node single node n1 to the graph
added node n1 to graph1

# Print the node in the graph
graph1 contains: n1
